package.json

```diff
@@ -1,48 +1,49 @@
-{
-  "name": "amplify-category-video",
-  "version": "3.1.1",
-  "description": "Plugin for Amplify to add support for live streaming. Made for Unicorn Trivia Workshop",
-  "main": "index.js",
-  "scripts": {
-    "postinstall": "node scripts/post-install.js",
-    "lint": "eslint .",
-    "lint-fix": "eslint . --fix"
-  },
-  "author": "wizage",
-  "license": "Apache 2.0",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/awslabs/amplify-video.git"
-  },
-  "keywords": [
-    "amplify",
-    "plugin",
-    "video"
-  ],
-  "dependencies": {
-    "archiver": "^3.1.1",
-    "chalk": "^2.4.2",
-    "child_process": "^1.0.2",
-    "ejs": "^2.7.1",
-    "fs-extra": "^7.0.0",
-    "ini": "^1.3.5",
-    "inquirer": "^6.5.0",
-    "mime-types": "^2.1.21",
-    "ora": "^4.0.3",
-    "sha1": "^1.1.1",
-    "yaml": "^1.7.2"
-  },
-  "devDependencies": {
-    "eslint": "^7.2.0",
-    "eslint-config-airbnb": "^18.2.0",
-    "eslint-config-airbnb-base": "^13.2.0",
-    "eslint-plugin-import": "^2.21.2",
-    "eslint-plugin-jsx-a11y": "^6.3.0",
-    "eslint-plugin-react": "^7.20.0",
-    "eslint-plugin-react-hooks": "^4.0.0"
-  },
-  "bugs": {
-    "url": "https://github.com/awslabs/amplify-video/issues"
-  },
-  "homepage": "https://github.com/awslabs/amplify-video#readme"
-}
+{
+  "name": "amplify-category-video",
+  "version": "3.1.1",
+  "description": "Plugin for Amplify to add support for live streaming. Made for Unicorn Trivia Workshop",
```

Comment:
we should update this description to something more accurate

Reply:
Yes! We should! Lets do it